### ISSUE TITLE
BT-633-Organization: Organisation Natural Person

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/european-union-support#135]
This field is a boolean to indicate whether an organization is a natural person, for example in the case of an award to a sole-trader.

In https://github.com/open-contracting/standard/issues/883#issuecomment-931311939, Colin proposed to discard this field. However, I can't see how else a user could determine whether an organization is a natural person or a legal person so it seems like that would result in information loss.

@jpmckinney is there a reason that we shouldn't add a `parties/details/naturalPerson` field to the [European Union extension](https://extensions.open-contracting.org/en/extensions/eu/)?

**Comment by jpmckinney** [comment]
partyScale has a code for 'selfEmployed': "This is a self-employed natural person, also known as a sole trader."

https://extensions.open-contracting.org/en/extensions/partyScale/master/codelists/

**Comment by duncandewhurst** [comment]
Ah, great. I didn't know about that code.

I checked the SDK documentation to determine whether the field can be used in a context other than a sole trader. It can be used in the context of ['non-tenderer, non-buyer' organizations](https://docs.ted.europa.eu/eforms/1.4.0/schema/all-in-one.html#_non_tenderer_non_buyer), which are described as:

> ...organizations active during the Public Procurement Procedure neither being a buyer nor a tenderer.

According to [Table 22. Mapping: Organization BTs — Schema components](https://docs.ted.europa.eu/eforms/1.4.0/schema/all-in-one.html#mappingOrganizationBTsSchemaComponentsTable), it is:

> Only for economic operators (tenderer, winner, service provider …​)

However, according to [Table 23. Information requirement per role](https://docs.ted.europa.eu/eforms/1.4.0/schema/all-in-one.html#informationRequirementPerRoleTable), it is an optional field for all roles, which seems contradictory. The field is conditionally mandatory for the following roles, which I think we can take as the only places it is intended to be used.

* Winner (#)
* Subcontractor
* Mediation organization
* Review organization (#)
* Procurement service provider

For any of the above roles, I think the mapping to 'selfEmployed' is appropriate so I've updated `guidance.yaml`.
